sys/include/net/gnrc/mac/types.h

```diff
@@ -46,6 +49,68 @@ typedef enum {
     TX_FEEDBACK_BUSY          /* found medium busy when doing transmission */
 } gnrc_mac_tx_feedback_t;
 
+/**
+ * @brief MAC internal type for storing reception state parameters and
+ *        state machines.
+ *        This structure can be extended to contain more needed
+ *        states and parameters. Please #ifdef+NEWMAC them out for particular
+ *        new MAC protocols.
+ */
+typedef struct {
+    /* RX queue for storing received packets */
+    gnrc_priority_pktqueue_t queue;
+
+    /* RX queue nodes */
+    gnrc_priority_pktqueue_node_t _queue_nodes[GNRC_MAC_RX_QUEUE_SIZE];
+
+    /* buffer for storing dispatching packets */
+    gnrc_pktsnip_t* dispatch_buffer[GNRC_MAC_DISPATCH_BUFFER_SIZE];
+} gnrc_mac_rx_t;
+
+/**
+ * @brief type for storing states of tx neighbor node.
+ */
+typedef struct {
+    uint8_t  l2_addr[IEEE802154_LONG_ADDRESS_LEN]; /* Address of neighbour node */
```

Comment:
Yes, the `gnrc_mac` module is specific to 802.15.4, currently. Should I reflect this '15.4' specification on the folder or module names?